packages/hebao_v1/contracts/base/MetaTxModule.sol

```diff
@@ -50,6 +50,8 @@ abstract contract MetaTxModule is BaseModule
     using SignatureUtil for bytes32;
     using BytesUtil     for bytes;
 
+    uint   constant public   BLOCK_BOUND = 100;
```

Comment:
This is pretty low, no? If the current block number at time of signing is put into the nonce than the transaction is only valid for around 20 minutes. If Ethereum is congested for even ~15 minutes people would have to re-sign their transactions.

Reply:
will 1000 be a reasonable value?

Reply:
I think that should be okay. Though on black Thursday Ethereum was congested (gas price > 50 Gwei) for almost a day I think? So in exceptional cases transactions could still expire I guess, but that may not be a bad thing.

Reply:
BLOCK_BOUND used for nonce not too large, limit the client don't fill the value too large.
It can not happen re-sign case.